go.mod

```diff
@@ -1,35 +1,35 @@
 module github.com/stackitcloud/gardener-extension-acl
 
-go 1.22.0
+go 1.22.4
 
-toolchain go1.22.5
+toolchain go1.22.8
 
 replace k8s.io/client-go => k8s.io/client-go v0.29.6
```

Comment:
Please try to replace the weird `client-go` version below and also bump it to v0.29.7

Reply:
I bumped `client-go` to `v0.29.7`.